pallets/committee/src/benchmarking.rs

```diff
@@ -0,0 +1,226 @@
+        .map(|i| user::<T>("member", i))
+        .collect();
+    Members::<I>::put(members.iter().map(|m| m.did()).collect::<Vec<_>>());
+    make_proposals_and_vote::<T, I>(&members)?;
+    Ok(members)
+}
+
+fn vote_verify<T, I>(
+    hash: <T as frame_system::Trait>::Hash,
+    ptoposal_num: u32,
+    vote: bool,
+) -> DispatchResult
+where
+    I: Instance,
+    T: Trait<I>,
+{
+    if COMMITTEE_MEMBERS_MAX > 4 || (COMMITTEE_MEMBERS_MAX == 4 && !vote) {
+        // The proposal is not finalised because there is no quorum yet.
+        if let Some(votes) = Voting::<T, I>::get(&hash) {
+            ensure!(votes.index == proposal_num, "wrong proposal_num");
+            if vote {
+                ensure!(votes.ayes.contains(&did), "aye vote missing");
+            } else {
+                ensure!(votes.nays.contains(&did), "nay vote missing");
+            }
+        } else {
+            return Err("cannot get votes".into());
+        }
+    } else {
+        // The proposal is finalised and removed from storage.
+        // TODO: pattern-match an event emitted during proposal finalisation.
+        ensure!(
+            !Voting::<T, I>::contains_key(&hash),
+            "executed proposal is not removed"
+        );
+    }
+}
+
+benchmarks_instance! {
+    _ {}
+
+    set_vote_threshold {
+        let n = 1;
+        let d = 2;
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_vote_threshold(n, d);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(Module::<T, _>::vote_threshold() == (n, d), "incorrect vote threshold");
+    }
+
+    set_release_coordinator {
+        let dids: Vec<_> = (0..COMMITTEE_MEMBERS_MAX).map(|i| user::<T>("member", i).did()).collect();
+        let coordinator = dids.last().unwrap().clone();
+        Members::<I>::put(dids);
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_release_coordinator(coordinator);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(
+            Module::<T, _>::release_coordinator() == Some(coordinator),
+            "incorrect release coordinator"
+        );
+    }
+
+    set_expires_after {
+        let maybe_block = MaybeBlock::Some(1.into());
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_expires_after(maybe_block);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(Module::<T, _>::expires_after() == maybe_block, "incorrect expiration");
+    }
+
+    vote_or_propose_new_proposal {
+        let members = make_members_and_proposals::<T, I>()?;
+        let last_proposal_num = ProposalCount::<I>::get();
+        let (proposal, hash) = make_proposal::<T, I>(0);
+        identity::CurrentDid::put(members[0].did());
+    }: vote_or_propose(members[0].origin.clone(), true, Box::new(proposal.clone()))
```

Comment:
Move out the clones into the setup phase; otherwise they'll count towards the weight.